addons/survey/views/survey_templates_statistics.xml

```diff
@@ -144,7 +118,61 @@
+                                <a t-attf-class="#{dropdown_item_classes} filter-all #{'active' if not search_finished else ''}"> All surveys
+                                    <span t-if="not search_finished" t-esc="survey_data['count_all']" class="badge badge-pill badge-primary ml-3"/>
+                                </a>
+                                <a t-attf-class="#{dropdown_item_classes} filter-finished #{'active' if search_finished else ''}"> Completed surveys
+                                    <span t-if="not search_finished" t-esc="survey_data['count_finished']" class="badge badge-pill badge-primary ml-3"/>
+                                </a>
+                            </div>
+                        </li>
+                        <li t-if="survey.scoring_type != 'no_scoring'" class="nav-item dropdown mr-2 my-1">
+                            <a href="#" role="button" t-attf-class="btn btn-outline-primary dropdown-toggle #{'active' if search_passed_or_failed else ''}" data-toggle="dropdown">
+                                <span t-if="search_failed">Failed surveys only</span>
+                                <span t-elif="search_passed">Passed surveys only</span>
+                                <span t-else="">Passed and failed surveys</span>
+                            </a>
+                            <div class="dropdown-menu">
+                                <a t-attf-class="#{dropdown_item_classes} filter-passed-and-failed #{'active' if not search_passed_or_failed else ''}"> Passed and failed surveys
+                                    <span t-if="not search_passed_or_failed" t-esc="survey_data['count_all']" class="badge badge-pill badge-primary ml-3"/>
+                                </a>
+                                <a t-attf-class="#{dropdown_item_classes} filter-passed #{'active' if search_passed else ''}"> Passed surveys only
+                                    <span t-if="not search_passed_or_failed" t-esc="survey_data['count_passed']" class="badge badge-pill badge-primary ml-3"/>
+                                </a>
+                                <a t-attf-class="#{dropdown_item_classes} filter-failed #{'active' if search_failed else ''}"> Failed surveys only
+                                    <span t-if="not search_passed_or_failed" t-esc="survey_data['count_failed']" class="badge badge-pill badge-primary ml-3"/>
+                                </a>
+                            </div>
+                        </li>
+                        <li t-foreach="search_filters" t-as="filter_data" class="nav-item mr-2 my-1">
+                            <span t-attf-class="btn btn-light"><span t-esc="filter_data['question']"></span> | <span t-esc="filter_data['answers']"></span>
+                                <i class="fa fa-times remove-question-filter" t-att-data-row_id="filter_data['row_id']" t-att-data-answer_id="filter_data['answer_id']"></i>
```

Comment:
Would rename the class to "filter-remove-question" to namespace all filter actions with "filter-".